packages/styles/src/switch/switch.scss

```diff
@@ -0,0 +1,80 @@
+@import "../core/_dev";
+
+.as-switch {
+  display: inline-block;
+  position: relative;
+
+  svg {
```

Comment:
We're not using BEM here, this selector could be `.as-switch--icon`. Not sure about the ones below, but I think we shouldn't be using element selectors as we agreed in other places.